mesonbuild/mesonmain.py

```diff
@@ -33,6 +33,10 @@ def create_parser():
     coredata.register_builtin_arguments(p)
     p.add_argument('--cross-file', default=None,
                    help='File describing cross compilation environment.')
+    p.add_argument('--config-file',
+                   default=[],
+                   action='append',
+                   help='File describing cross compilation environment.')
```

Comment:
The help string looks like a copy and paste error. If not some more explanation what the difference to `--cross-file` is, would be nice.